paimon-core/src/main/java/org/apache/paimon/io/DataFileMeta.java

```diff
@@ -58,7 +60,13 @@ public class DataFileMeta {
 
     private final String fileName;
     private final long fileSize;
-    private final long rowCount;
+
+    // totalRowCount = addRowCount + deleteRowCount
+    // Why don't we keep addRowCount and deleteRowCount?
+    // Because in previous versions of DataFileMeta, we only keep rowCount.
+    // We have to keep the compatibility.
+    private final long totalRowCount;
```

Comment:
Can we still use `rowCount` naming?
Because in schema, it is still `_ROW_COUNT`...